src/pyuvdata/utils/io/ms.py

```diff
@@ -1358,17 +1374,58 @@ def read_ms_feed(filepath):
         If no MS file is found with the provided name.
     """
     _ms_utils_call_checks(filepath + "/FEED")
+    filepath += "::FEED"
 
-    raise NotImplementedError("Reading of MS FEED tables not available yet.")
+    with tables.table(filepath, ack=False) as feed_table:
+        if "pyuvdata_has_feed" in feed_table.getkeywords() and not (
+            feed_table.getkeyword("pyuvdata_has_feed")
+        ):
+            feed_array = feed_angle = Nfeeds = antenna_numbers = None
```

Comment:
What is this handling? Does this mean that this file was written by pyuvdata?

Reply:
Yes -- written by pyuvdata, but _before_ we necessarily had correct handling of the feed information. I don't know that this is the correct way to handle this, but I did at least want to catch when this information might be at least suspect.